### ISSUE TITLE
Crash when invoking liara create-config

### ISSUE THREAD

**Issue by rys** [Anteru/liara#1]
When using `liara create-config` to generate a default `config.yaml`, it crashes because when it calls `dump_yaml()`, `options.output` is just a string with the filename, not a file object that Python can call `write()` on.

Stacktrace:

```
Traceback (most recent call last):
  File "/usr/local/bin/liara", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.7/site-packages/liara/cmdline.py", line 96, in cli
    args.func(args)
  File "/usr/local/lib/python3.7/site-packages/liara/cmdline.py", line 191, in create_config
    dump_yaml(create_default_configuration(), options.output)
  File "/usr/local/lib/python3.7/site-packages/liara/yaml.py", line 27, in dump_yaml
    return yaml.dump(data, stream, Dumper=Dumper)
  File "/usr/local/lib/python3.7/site-packages/yaml/__init__.py", line 290, in dump
    return dump_all([data], stream, Dumper=Dumper, **kwds)
  File "/usr/local/lib/python3.7/site-packages/yaml/__init__.py", line 278, in dump_all
    dumper.represent(data)
  File "/usr/local/lib/python3.7/site-packages/yaml/representer.py", line 28, in represent
    self.serialize(node)
  File "ext/_yaml.pyx", line 1350, in _yaml.CEmitter.serialize
  File "ext/_yaml.pyx", line 1525, in _yaml.output_handler
```

No patch to fix since I'm not sure where the right place to put the open with the right access mode is.

**Comment by Anteru** [comment]
Thanks, I'll take a look!

**Comment by Anteru** [comment]
Fixed in f2406484e0f63a54d1d9d09545941f7fac7fafb3

**Comment by rys** [comment]
Looks good, didn't know `argparse` could do that!
